### DISCUSSION TITLE
[Feature] confirmation for delete and trash

### DISCUSSION THREAD

**Discussion by Ai-Kiwi** [immich-app/immich#14207] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

When you click to move an item to the trash or to archieve it it should prompt asking if you are sure you want to delete it. I keep finding items I own have been achieved as on mobile it is very easy to accidentally click achieve instead of exit out of app. This could be added as a setting as well for those who don't like it so it doesn't annoy them. 

### Platform

- [ ] Server
- [X] Web
- [X] Mobile

**Comment by RenaKunisaki** [comment]
Very much need either this, or the ability to sort trash by recently deleted, to be able to find the file I just accidentally deleted.

**Comment by bo0tzz** [comment]
#5864
